### DISCUSSION TITLE
After-Response Script

### DISCUSSION THREAD

**Discussion by smiller0924** [Kong/insomnia#7463] [Feedback]
I just want to say that the After-Response Script is awesome!  I was able to:
- create variables to build a count for loop.
- capture Base64 encoded Zebra ZPL labels in the API Response.
- Run an atob on the variable.
- Feed the ZPL into a Labelrary API to display the labels in Insomnia's Preview Window.

I was even able to copy my postman scripts into the pre- and post-script functions flawlessly.

If I had one request it would be to have the pre-/post-scripting at folder levels. This way I don't have to keep putting the script in every API call I need it for. Just loading it once would better. This way if I have to make a change it can cascade across all my API calls. 

All I gotta is,"best feature enhancement yet!"

Keep it up;👍

**Comment by ihexxa** [comment]
Thanks @smiller0924 for the feedback, I believe that folder-level scripting will come soon, cc @subnetmarco.

**Comment by subnetmarco** [comment]
Folder-level scripting is coming as part of 9.3 GA, so when "After response scripting" is GA, the folder-level scripting will be there as well.

**Reply by smiller0924** [reply]
I hate to ask. What is 'GA'?

**Reply by subnetmarco** [reply]
General Availability

**Reply by smiller0924** [reply]
Thank you for clarifying.

**Comment by subnetmarco** [comment]
Since this capability is coming in 9.3, I am closing this discussion for the time being.
